ARICHRecoUtils/classes_def.xml

```diff
@@ -0,0 +1,17 @@
+<lcgdict>
+
+<class name="ARICHRECO::ARICH_UTILS"/>
+<class name="ARICHRECO::Arich"/>
+<class name="ARICHRECO::Aerogel"/>
+<class name="ARICHRECO::Beam"/>
+<class name="ARICHRECO::Detector"/>
+<class name="ARICHRECO::Particle"/>
+<class name="ARICHRECO::Photon"/>
+<class name="ARICHRECO::HoughFitter"/>
```

Comment:
ah all of these new classes are going to want version take a look here for example:
https://github.com/EMPHATICSoft/emphaticsoft/blob/main/DataQuality/classes_def.xml
standard practice is to start at version 10